uint64 serial # Serial id from virtual traffic simulation

float64 sim_t # Simulation time of the scenario

bool sim_start # Whether the simulation has started

float64[40] front_s # Vehicle position in time sequence
float64[40] front_v # Vehicle speed in time sequence
float64[40] front_a # Vehicle acceleration in time sequence